editor/plugins/text_control_editor_plugin.cpp

```diff
@@ -301,47 +292,46 @@ void TextControlEditor::_set_font() {
 	for (int i = 0; i < count; ++i) {
 		Control *edited_control = edited_controls[i];
 
-		String edited_font;
-		if (edited_control->is_class("RichTextLabel")) {
-			edited_font = "normal_font";
+		StringName edited_font;
+		if (Object::cast_to<RichTextLabel>(edited_control)) {
+			edited_font = SNAME("normal_font");
 		} else {
-			edited_font = "font";
+			edited_font = SNAME("font");
 		}
 
 		if (font_list->get_selected_id() == FONT_INFO_THEME_DEFAULT) {
 			// Remove font override.
-			ur->add_do_method(edited_control, "remove_theme_font_override", edited_font);
+			ur->add_do_method(edited_control, SNAME("remove_theme_font_override"), edited_font);
```

Comment:
Following #57788 it seems like using `SNAME` for UndoRedo methods would not be recommended, so I guess this part could be undone (the rest seems fine).

Reply:
Done.